ui/src/app/auth/signup/page.tsx

```
@@ -0,0 +1,40 @@
+          placeholder="Email"
+          required
+        />
+        <input
+          type="password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          placeholder="Password"
+          required
+        />
+        <button type="submit" disabled={false}>
+          {false ? "Signing Up..." : "Signup"}
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)